src/pudl_rmi/coordinate.py

```diff
@@ -0,0 +1,247 @@
+"""
+Coordinate the acquisition and generation of RMI's interrelated outputs.
+
+The outputs in this repo are depended on one another. See `README` for a
+diagram of the realtions.
+"""
+
+import logging
+import pandas as pd
+
+import pudl_rmi
+
+logger = logging.getLogger(__name__)
+
+
+class Output():
+    """Class to manage all of the interconnected RMI outputs."""
+
+    def __init__(self, pudl_out):
+        """
+        Initialize output coordinator for the RMI ouptus.
+
+        This object enables short term disk caching and an easy way to
+        regenerate any or all of the interconnected RMI outputs. Each `get_`
+        method in this object grabs a pickle file off disk if it is available,
+        or generates the output if it is not available and/or if you set
+        clobber to True.
+
+        Because some outputs rely on others, this object enables clobbering of
+        both the main output and the outputs which the main object relies on.
+        This in effect enables a clobber and deep clobbers.
+
+        Most of the outputs are generated via a ``execute()`` function from the
+        cooresponding module for that output. The plant-part list is generated
+        from the `pudl_out` object.
+
+        Args:
+            pudl_out (object): instance of `pudl.output.pudltabl.PudlTabl()`.
+                The frequency (`freq`) of `pudl_out` must be `AS`. For best
+                results  `fill_fuel_cost`, `roll_fuel_cost`, and `fill_net_gen`
+                should all be True.
+        """
+        self.pudl_out = pudl_out
+        if pudl_out.freq != 'AS':
+            raise AssertionError(
+                f"Frequency of `pudl_out` must be `AS` but got {pudl_out.freq}"
+            )
+
+    def get_plant_part_list(self, clobber=False):
+        """
+        Get the master unit list; generate it or get if from a file.
+
+        If you generate the MUL, it will be saved at the file path given. The
+        plant-parts list is generated via the pudl_out object.
+
+        Args:
+            clobber (boolean): True if you want to regenerate the master unit
+                list whether or not the output is already pickled. Default is
+                False.
+        """
+        if not pudl_rmi.FILE_PATH_PLANT_PARTS_EIA.is_file() or clobber:
```

Comment:
I get the sense from looking at other peoples' code that the norm is to import global names directly and use them without their parent module, if you want to shorten a bunch of these references by doing

```py
from pudl_rmi import FILE_PATH_PLANT_PARTS_EIA, FILE_PATH_DEPRISH, ...
```

up top.